Packs/Slack/Integrations/Slack_IAM/README.md

```diff
@@ -0,0 +1,475 @@
+Note: This integration should be used as part of our **Identity Lifecycle Management** premium pack. For further details, visit our IAM pack documentation.
+
+Integrate with Slack's services to execute CRUD operations for employee lifecycle processes.
+For more information, please refer to the [Identity Lifecycle Management article](https://xsoar.pan.dev/docs/reference/articles/identity-lifecycle-management).
+
+## Configure Slack IAM on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Slack IAM.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | access_token | Access Token | True |
+    | insecure | Trust any certificate \(not secure\) | False |
+    | proxy | Use system proxy settings | False |
+    | create_user_enabled | iam-create-user Command Enabled | True | If set to false, iam-create-user command will be skipped, and user will not be created.
+    | update_user_enabled | iam-update-user Command Enabled | True |
+    | disable_user_enabled | iam-disable-user Command Enabled | True |
+    | create_if_not_exists | Automatically create user if not found in update command | True |
```

Comment:
my mistake, True/False is regarding whether these fields are required or not. It was correct when it was False.
Regarding the `If set to false, iam-create-user command will be skipped, and user will not be created.`
That is in the wrong place, you will not see it in the markdown table.
```suggestion
    | create_user_enabled | iam-create-user Command Enabled | True |
    | update_user_enabled | iam-update-user Command Enabled | True |
    | disable_user_enabled | iam-disable-user Command Enabled | True |
    | create_if_not_exists | Automatically create user if not found in update command | True |
```